menyer/enmeny/enmeny.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="sv">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style1.css">
+    <title>En av alla menyer</title>
+</head>
+
+<body>
+    <center>
+        <header class="header">
+            <div class="enmeny">
+                <h1>
+                    &#9731; En meny med god mat och en del helt ok drycker! &#9731;
+                </h1>
+            </div><br><br><br>
```

Comment:
Avoid using <br> tag for making space. Consider using magin-bottom styling instead.